packages/tooling/fast-tooling/src/data-utilities/mapping.mdn-data.types.ts

```diff
@@ -0,0 +1,361 @@
+/**
+ * Referenced from: https://github.com/mdn/data/blob/master/index.js
+ * This should be added to DefinitivelyTyped in future:
```

Comment:
Do you know if there are there any issues / pull-requests tracking this that can be referenced?

Reply:
Not that I could find, I've only specified the types that matter for my purposes right now and I'll be expanding on them as needed, then later I'll add them to DefinitivelyTyped.